index.html

```diff
@@ -1,20 +1,288 @@
+				</a>
+			</div>
+		</div>
+		<!-- navigation menu  -->
+		<nav class="navbar relative bg-transparent md:bg-white md:shadow-sm">
+			<div class="navbar-inner mx-auto flex items-center justify-between gap-4 py-3 md:w-10/12">
+				<button id="hamburger-button" class="hamburger flex-start bg h-10 cursor-pointer md:hidden">
+					<i class="fa-sharp fa-solid fa-bars" style="font-size: 34px"></i>
+				</button>
+
+				<!-- logo  -->
+				<div class="hide--mobile">
+					<a href="index.html" class="logo items-center gap-2 leading-6 transition-all md:flex">
+						<img src="./public/images/logo-arabic2.jpg" style="width: 144px" alt="logo" />
+					</a>
+				</div>
+
+				<div class="relative flex items-center justify-center">
+					<ul class="hidden items-center gap-9 gap-y-4 text-xl md:flex md:flex-wrap" aria-label="main">
+						<li class="text-coral-red hover:text-coral-red hover:opacity-90">
+							<a href="./about.html" class=""> About </a>
+						</li>
+						<li class="hover:text-coral-red hover:opacity-90">
+							<a href="#program" class=""> Program </a>
+						</li>
+
+						<li class="hover:text-coral-red hover:opacity-90">
+							<a href="#join" class=""> Join </a>
+						</li>
+
+						<li class="hover:text-coral-red hover:opacity-90">
+							<a href="#partner" class=""> Sponsor </a>
+						</li>
+						<li class="hover:text-coral-red hover:opacity-90">
+							<a href="#" class=""> News </a>
+						</li>
+						<li
+							class="border-2 border-coral-red py-2 px-4 transition-all hover:bg-coral-red hover:text-white hover:opacity-90">
+							<a href="index.html" class=""> CC Campaign </a>
+						</li>
+					</ul>
+				</div>
+			</div>
+			<!-- Mobile menu -->
+			<div id="mobile-menu"
+				class="mobile-menu absolute top-0 hidden w-full origin-top animate-times-menu-animation flex-col justify-center bg-metal px-4 py-3 text-4xl text-white">
+				<button class="close self-start">X</button>
+				<div class="flex h-screen-94h flex-col items-center gap-4 py-8" aria-label="mobile-menu">
+					<a href="./about.html" class="w-full nav-link px-4 text-center hover:opacity-70">About</a>
+					<a href="#program" class="nav-link w-full px-4 text-center hover:opacity-70">Program</a>
+					<a href="#" class="nav-link w-full px-4 text-center hover:opacity-70">Join</a>
+					<a href="#" class="nav-link w-full px-4 text-center hover:opacity-70">Sponsor</a>
+					<a href="#" class="nav-link w-full px-4 text-center hover:opacity-70">News</a>
+				</div>
+			</div>
+		</nav>
+	</header>
+
+	<div
+		class="container relative h-full bg-cover bg-no-repeat hero-area mx-auto flex flex-col gap-4 px-4 py-28 text-metal-grey">
+		<p class="hero-big text-2xl font-medium text-coral-red" style="color: #ec5242">
+			"Hello! Sharing world"
+		</p>
+		<h1 class="hero-container bg-cover bg-clip-text text-3xl font-bold uppercase text-transparent min-w-680px text-6xl">
+			World Conference <br />
+			Arabic Language Day <br class="hide--mobile" />
+			2022
+		</h1>
```

Comment:
- Good job for fulling the previous core reviewer suggestions 👍👍, I tink you have missed one previous reviewer's suggestion here.👌 
- Please try to make pages look almost identical to the original design, outside the content differences with the exact same colors, for text, backgrounds, borders, etc, from the original design.


| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/73607512/210773953-06557d35-8dd5-4607-a79c-ceefac72f9a3.PNG" width="200">|<img src="https://user-images.githubusercontent.com/73607512/210774426-d27bae82-d00e-4f36-895d-5ec47e0b6dad.png" width="200">|


😎️

🤝️
* Basic color: #272A31, #EC5242, #D3D3D3